lib/LanguageServerCompletion/Handler/CompletionHandler.php

```diff
@@ -111,16 +111,25 @@ public function completion(CompletionParams $params, CancellationToken $token):
                         : InsertTextFormat::PLAIN_TEXT
                     ;
                 }
-                
-                $items[] = $i = CompletionItem::fromArray([
+
+                $importNameResult = $this->importClassOrFunctionName($suggestion, $params);
+
+                if ($importNameResult->isSuccessAndHasAliasedNameImport()) {
+                    // todo is this the right place?
```

Comment:
My feelings say this is the wrong place and way for doing this.  
See the "text edit conversion" comment below as well.